examples/java/spring-boot/src/test/java/uk/gov/api/springboot/application/services/EmptyFetcherServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.api.springboot.application.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.api.springboot.domain.model.RegistryEntry;
+
+class EmptyFetcherServiceTest {
+
+  @Test
+  void fetchReturnsAnEmptyList() {
+    var fetcherService = new EmptyFetcherService();
```

Comment:
Although not Rule of Three, I'd say we should make this a `final` field - same with other options